libraries/src/test/java/com/baeldung/mapdb/SortedTableMapUnitTest.java

```diff
@@ -0,0 +1,50 @@
+package com.baeldung.mapdb;
+
+import org.junit.Test;
+import org.mapdb.Serializer;
+import org.mapdb.SortedTableMap;
+import org.mapdb.volume.MappedFileVol;
+import org.mapdb.volume.Volume;
+
+import static junit.framework.Assert.assertEquals;
+
+public class SortedTableMapUnitTest {
+
+  @Test
+  public void givenValidSortedTableMapSetup_whenQueried_checkValuesCorrect() {
+
+    final String VOLUME_LOCATION = "sortedTableMapVol.db";
```

Comment:
this looks like it would rather be `private static final`, which would be good.

Reply:
moved `VOLUME_LOCATION` into a `private static final` field.